### PR TITLE
Added scripts to convert from .muse format to the combined.csv format

### PR DESCRIPTION
**Pull request by mtriff**
Script usage
-----------------
For one file:
```
./convert.sh muselab_recording.muse
```

For an entire directory:
```
./convertAll.sh /Applications/Muse/recordings
```

### CODE REVIEW

**Comment by Glavin001** [comment]
Thank you, @mtriff ! This is great, we can use the Muse tools and convert directly to the desired CSV format.

**Comment by mtriff** [comment]
My pleasure!

**Comment by Glavin001** [comment]
So the files that are being created are incomplete. The header is 20 columns while the data in the rows is only 16 columns:

```
Alpha - Location 1,Alpha - Location 2,Alpha - Location 3,Alpha - Location 4,Beta - Location 1,Beta - Location 2,Beta - Location 3,Beta - Location 4,Delta - Location 1,Delta - Location 2,Delta - Location 3,Delta - Location 4,Gamma - Location 1,Gamma - Location 2,Gamma - Location 3,Gamma - Location 4,Theta - Location 1,Theta - Location 2,Theta - Location 3,Theta - Location 4
0.43705958,0.037856277,0.16795437,0.18538538,0.10492487,0.023092046,0.13551691,0.23223358,0.19599223,0.82287997,0.33475035,0.20498703,0.22042431,0.089140475,0.28653625,0.3105342
```

File: src/data-collection/convert.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+muse-player -f "$1" -i /muse/elements/alpha_relative -C alpha_rel_out.csv
+muse-player -f "$1" -i /muse/elements/beta_relative -C beta_rel_out.csv
+muse-player -f "$1" -i /muse/elements/delta_relative -C delta_rel_out.csv
+muse-player -f "$1" -i /muse/elements/theta_relative -C theta_rel_out.csv
+
```
Comment (Glavin001) [inline src/data-collection/convert.sh]: Looks like we are missing Gamma wave.

**Comment by Glavin001** [comment]
@mtriff would you be able to add the Gamma wave and re-convert the original .muse files to the .csv, after you are finished recording your next sessions?

**Comment by mtriff** [comment]
Good catch, thanks. I'll re-convert ASAP.

**Pull request merged**